invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/deposit.less

```diff
@@ -129,3 +128,29 @@
 .item div.description:not(.license-item-description) {
   display: initial !important;
 }
+
+.deposits-community-header {
+  margin: -20px 0px 20px 0px;
+  background-color: #ededed;
+  height: 65px;
+}
+
+.community-header {
+  padding-top: 10px;
```

Comment:
what about using `.relaxed` class of container ?

Reply:
Tried to use it but seems not to work

Reply:
then I would apply the generated padding classes from invenio theme `pt-10`

Reply:
Hmm I think in this particular case it's more handy have the rule defined in this class, mainly because `community-header` class is used widely and if we want to change it we can easily change it only in this class, otherwise we would need to go to all the tags and change the `pt-XX` and we would probably forget some.